lib/improver/tests/ensemble_calibration/ensemble_calibration/test_EnsembleCalibration.py

```diff
@@ -599,22 +359,27 @@ def test_temperature_realizations_data_check(self):
         The ensemble realizations is the predictor.
         """
         distribution = "gaussian"
-        desired_units = "degreesC"
         predictor_of_mean_flag = "realizations"
         plugin = Plugin(
-            self.calibration_method, distribution, desired_units,
+            self.calibration_method, distribution,
             predictor_of_mean_flag=predictor_of_mean_flag,
-            max_iterations=300)
+            max_iterations=15)
         calibrated_predictor, calibrated_variance = plugin.process(
             self.current_temperature_forecast_cube,
             self.historic_temperature_forecast_cube,
             self.temperature_truth_cube)
         self.assertArrayAlmostEqual(
             calibrated_predictor.data,
-            self.expected_temperature_predictor_data, decimal=3)
+            self.expected_specific_temperature_predictor_data)
+        self.assertArrayAlmostEqual(
+            calibrated_variance.data,
+            self.expected_specific_temperature_variance_data)
+        self.assertArrayAlmostEqual(
+            calibrated_predictor.data,
+            self.expected_temperature_predictor_data, decimal=0)
```

Comment:
How has it been decided that outputs that differ at the first decimal place will be acceptable from this scheme?

Reply:
Having discussed with @bjwheltor it seems like this is a sensible idea for the moment, to get the tests working in master.  There are several positive things in this PR so I'm tending towards this rather than reverting the old PR.  However, whatever gets master fixed quickest under the circumstances is probably the way to go.